app/views/users/confirm_destroy.html.erb

```diff
@@ -1,55 +1,59 @@
-      const deleteAccountUsernameInput = document.getElementById('delete__account__username__field');
-      const deleteAcccountVerificationInput = document.getElementById('delete__account__verification__field');
-      function bothInputsVerified() {
-        return deleteAccountUsernameInput.value === '<%= @user.username %>' && deleteAcccountVerificationInput.value === "delete my account";
-      }
-      const disableDeleteAccountBtn = () => { deleteAccountBtn.disabled = !bothInputsVerified() };
-      deleteAccountUsernameInput.addEventListener('input', disableDeleteAccountBtn);
-      deleteAcccountVerificationInput.addEventListener('input', disableDeleteAccountBtn);
-    </script>
-    <p>
-      If you would like to keep your content under the <%= link_to "@ghost", "/ghost" %> account,
-      please <%= email_link(text: "click here", additional_info: { subject: "Request Account Deletion", body: @email_body }) %>
-    </p>
+<div class="user-settings-page crayons-layout crayons-layout--limited-l gap-0">
+  <div class="crayons-card min-w-0 text-padding">
+    <h2 class="crayons-title"> Destroy your account</h2>
+    <div class="settings-form">
+      <h4 class="fw-normal">
+        <%= form_tag user_full_delete_path, method: :delete, autocomplete: "off", id: "delete__account" do %>
+          Deleting your account will:
+          <ul class="delete__account mb-4">
+            <li>delete your profile, along with your Twitter and/or GitHub associations.
+              This does not include app permissions. You will have to remove them yourself on
+              <a href="https://twitter.com/settings/applications">Twitter profile settings</a> and
+              <a href="https://github.com/settings/applications">GitHub profile settings</a>.
+            </li>
+            <%# TODO: expand the delete messaging later %>
+            <li>delete any and all content you have, such as articles, comments, your reading list or chat messages.</li>
+            <li>allow your username to become available to anyone.</li>
+          </ul>
+          <div class="field mb-4 w-50">
+            <label for="delete__account__username__field" class="crayons-field__label">
+              To delete your account:
+              <p class="crayons-field__description">type in your username</p>
+            </label>
+            <input type="text" name="delete__account__username__field" class="crayons-textfield " id="delete__account__username__field" placeholder="username">
+          </div>
+          <div class="field mb-4 w-50">
+            <label for="delete__account__verification__field" class="crayons-field__label">
+              For verification, type the words:
+              <p class="crayons-field__description"> delete my account</p>
+            </label>
+            <input type="text" name="delete__account__verification__field" class="crayons-textfield" id="delete__account__verification__field">
+          </div>
```

Comment:
```suggestion
        <div class="crayons-field m:w-50">
          <label for="delete__account__username__field" class="crayons-field__label">
            Type in your username
          </label>
          <input type="text" name="delete__account__username__field" class="crayons-textfield " id="delete__account__username__field" placeholder="username">
        </div>
        <div class="crayons-field m:w-50">
          <label for="delete__account__verification__field" class="crayons-field__label">
            For confirmaiton, type the words: <code>delete my account</code>
          </label>
          <input type="text" name="delete__account__verification__field" class="crayons-textfield" id="delete__account__verification__field">
        </div>
```

- this is how `.crayons-field` component should be used with pure HTML so it does render properly
- I've also suggested some changes to the copy as well.